docs/developer/decision-records/2022-06-08-identity-hub/authorization.puml

```diff
@@ -17,6 +21,9 @@ group authorize request
   B -> B: Verify JWS
   B -> IDHUB ++: Get Verifiable Presentation
   return Verifiable Presentation
+  B -> DA ++: Get issuer DID Document
```

Comment:
I would consider adding a quick description about this in the "Participant Authorization" paragraph.
When reading the doc it is not so obvious why we get the did of the Authority.

Reply:
Done